updated content and added release notes for MA-846 (#4795)
Kong/docs.konghq.com: app/konnect/analytics/generate-reports.md

1. Choose a time frame to display.

    Select between **Relative** or **Custom** to set any date range.

>     **Relative** time frames are dynamic and the report will capture a snapshot of data
>     relatively from the time a user views the report.
> 
>     **Custom** time frames are static and the report will capture a snapshot of data
>     in whichever time frame you choose. You can see a preview of the exact range below

Reviewer: ```suggestion
    **Relative** time frames are dynamic and the report captures a snapshot of data
    relative to when a user views the report.

    **Custom** time frames are static and the report captures a snapshot of data
    during the specified time frame. You can see a preview of the exact range below
```

    the time frame selector. For example:

    ```
    Sunday, Jun 12, 9:57:45 AM (CEST) to Monday, Jun 13, 9:57:45 AM (CEST)
    ```

> 1. Choose an entity type.

Reviewer: Is this being renamed in the UI (because the field is currently "Report type")?

Author: Yes.

## Metrics

Metric | Description
-------|------------
Request count | Total number of API calls within the selected time frame.
Requests per minute | Number of API calls per minute within the selected time frame.

> Response latency | The amount of time, in milliseconds, that it takes to process an API request. The time starts when the Kong Gateway receives a request and ends when it forwards the response back to the original caller. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 

Reviewer: ```suggestion
Response latency | The amount of time, in milliseconds, that it takes to process an API request. The time starts when {{site.base_gateway}} receives a request and ends when it forwards the response back to the original caller. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response latency of 10 milliseconds means that every 1 in 100 requests took at least 10 milliseconds from request received until response returned. 
```

Request size | The size of the request payload received from the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile request size of 100 bytes means that the payload size for every 1 in 100 requests was at least 100 bytes.
Response size | The size of the response payload returned to the client, in bytes. Users can select between different percentiles (p99, p95, and p50). For example, a 99th percentile response size of 100 bytes means that the payload size for every 1 in 100 response back to the original caller was at least 100 bytes.

{:.note}

> > **Note:** {{site.konnect_saas}} Analytics Analytics is using percentiles to enable users to understand the real performance characteristics of their APIs. They have the advantage of distinguishing users who have a good experience from those that have a bad experience. Percentiles can show you a more accurate picture of what most users experience using your API as opposed to hiding critical experiences in an average.

Reviewer: ```suggestion
> **Note:** {{site.konnect_saas}} Analytics uses percentiles to enable you to understand the real performance characteristics of your APIs. Percentiles depict a more accurate picture of what most end users experience using your API instead of hiding critical experiences by displaying averages.
```
